cheroot/errors.py

```diff
@@ -64,15 +72,27 @@ def plat_specific_errors(*errnames):
     socket_errors_to_ignore.extend(plat_specific_errors('EPROTOTYPE'))
     socket_errors_nonblocking.extend(plat_specific_errors('EPROTOTYPE'))
 
+sock_shutdown_errs = (OSError, _OpenSSL_SysCallError)
+write_shutdown_errs = (OSError, _OpenSSL_SysCallError, ConnectionError)
 
 acceptable_sock_shutdown_error_codes = {
+    errno.EBADF,
     errno.ENOTCONN,
     errno.EPIPE,
     errno.ESHUTDOWN,  # corresponds to BrokenPipeError in Python 3
     errno.ECONNRESET,  # corresponds to ConnectionResetError in Python 3
 }
-"""Errors that may happen during the connection close sequence.
 
+if IS_WINDOWS:
+    # Add Windows socket error code constant
```

Comment:
This phrase just repeats what's already in code. Code comments are supposed to explain motivation, not duplicate what's right below.